auth/token-managers/container-token-manager.ts

```diff
@@ -115,7 +115,7 @@ export class ContainerTokenManager extends IamRequestBasedTokenManager {
   /**
    * Request an IAM token using a compute resource token.
    */
-  protected async requestToken(): Promise<any> {
```

Comment:
This must have been a typo or copy/paste error. It doesn't change behavior whatsoever, the function is still "asynchronous", but we don't use the `await` keyword in this function so it's unnecessary to include the `async` keyword (it's only purpose is to provide context when using `await`).